compiler/ballerina-backend-jvm-old/src/main/ballerina/src/compiler_backend_jvm/interop/external_method_gen.bal

```diff
@@ -192,6 +192,12 @@ function createOldStyleExternalFunctionWrapper(bir:Function birFunc, string orgN
     addDefaultableBooleanVarsToSignature(birFunc);
     bir:BInvokableType functionTypeDesc = birFunc.typeValue;
 
+    bir:BType? restType = functionTypeDesc?.restType;
+
+    if (!(restType is ())) {
```

Comment:
I feel it would increase readability if you use restType is bir:BType instead of the negated statement? WDYT
```suggestion
    if ( restType is bir:BType ) {
```

Reply:
@irshadnilam @dulvinw I actually have a different opinion.  Checking for the union type will have a performance impact on the compile time. Its much faster check for nil and negating it than checking for multiple types (members of union). Refer https://github.com/ballerina-platform/ballerina-lang/pull/19197

Reply:
@dulvinw @rdhananjaya 
This seems to be the practice i saw in existing implementations of ballerina-backend. Also taking @SupunS consideration, i think we should keep it as it is. WDYT?

Reply:
+1 to keeps it if it's a common practice in JBal codegen side.